CONTRIBUTING.md

```diff
@@ -113,6 +113,32 @@ If you'd like to contribute example `.bicep` files that showcase abilities of th
 
 **Note:** If you have never submitted a Pull Request or used git before, reading through the [Git tutorial](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/git-tutorial.md) in the azure-quickstart-template repo is a good place to start.
 
+### Snippets
+
+If you'd like to contribute to the collection of snippets:  
+
+* A snippet should be a single, generic, resource. Ensure your snippet meets this criteria.
+* Add a Bicep file here, using the naming convention `res-RESOURCENAME.bicep`: [`./src/Bicep.LangServer/Snippets/Templates`](./src/Bicep.LangServer/Snippets/Templates)
+  * The file name without extension will be used as the label.
+  * A single line comment at the top of the file will be used as the description.
+  * E.g. [`res-aks-cluster.bicep`](./src/Bicep.LangServer/Snippets/Templates/res-aks-cluster.bicep) results in the following label and description:
+ ![](./docs/images/snippet-template.png)
+
+* Add a new folder in the following directory, for an integration test that validates snippet completion: [`./src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates`](./src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates)
```

Comment:
Is it possible for the snippet definition at `Bicep.LangServer/Snippets/Templates` be located in the same directory as this "values" bicep file? I guess the tests will catch it, but I worry that a lot of people will miss this on the first go.

Reply:
Will be included in https://github.com/Azure/bicep/issues/2247